web/packages/teleport/src/TrustedClusters/index.ts

```diff
@@ -17,4 +17,4 @@
  */
 
 import TrustedClusters from './TrustedClusters';
-export default TrustedClusters;
+export { TrustedClusters };
```

Comment:
Can we use this PR to clean these up all the way through to the component?

That way we can have a consistent

```typescript
export { TrustedClusters } from './TrustedClusters';
```

I know it's slightly increasing the scope of this PR, but we'd never get round to the cleanup otherwise

Feel free to ignore